lib/components/user/places/place-editor.tsx

```diff
@@ -64,15 +64,18 @@ class PlaceEditor extends Component<
     values: any
   } & WrappedComponentProps
 > {
-  _handleLocationChange = ({
-    location
-  }: {
-    location: {
-      lat: number
-      lon: number
-      name: string
+  _handleLocationChange = (
+    _: unknown, // Ignore intl object. TODO: localize name?
+    {
+      location
+    }: {
+      location: {
```

Comment:
For `location`, you can use the `Location` type from `@opentripplanner/types`.

Reply:
Added in 71c5a88!